docs/reference/mapping/types/flattened.asciidoc

```diff
@@ -307,3 +307,123 @@ The following mapping parameters are accepted:
     Whether <<full-text-queries,full text queries>> should split the input on
     whitespace when building a query for this field. Accepts `true` or `false`
     (default).
+
+[[flattened-synthetic-source]]
+==== Synthetic `_source`
+
+IMPORTANT: Synthetic `_source` is Generally Available only for TSDB indices
+(indices that have `index.mode` set to `time_series`). For other indices
+synthetic `_source` is in technical preview. Features in technical preview may
+be changed or removed in a future release. Elastic will apply best effort to fix
+any issues, but features in technical preview are not subject to the support SLA
+of official GA features.
+
+Flattened fields support <<synthetic-source,synthetic`_source`>> in their default
+configuration. Synthetic `_source` cannot be used with <<doc-values,`doc_values`>>
+disabled.
+
+Synthetic source always sorts alphabetically and de-duplicates keyword fields.
+For example:
+[source,console,id=synthetic-source-flattened-sorting-example]
+----
+PUT idx
+{
+  "mappings": {
+    "_source": { "mode": "synthetic" },
+    "properties": {
+      "keywords": { "type": "flattened" }
+    }
+  }
+}
+PUT idx/_doc/1
+{
+  "keywords": {
```

Comment:
This should probably be `flattened` or `flags` or something. Not `keywords` because it isn't of type `keyword`.